### DISCUSSION TITLE
如何一对多发款？

### DISCUSSION THREAD

**Discussion by wwaayyaa** [nervosnetwork/ckb#2403] [General]
看起来transfer只能指定一个收款人和一笔金额？

**Comment by wwaayyaa** [comment]
顺便再问一下，连续发款会有确认数限制吗？是否可以0确认一直发？

**Comment by keroro520** [comment]
是指 ckb-cli 的 `transfer` 命令吗？

**Comment by doitian** [comment]
一个 tx 里放多个 outputs 给不同的收款人就可以了，因为是一个交易不存在需要等确认。

**Comment by doitian** [comment]
交易使用 tx hash 引用，可以引用还没上链和还没确认的交易。不过这样的链式交易一旦一个依赖最终没上链，后面的交易也全部都上不了链了。

**Comment by keroro520** [comment]
> 顺便再问一下，连续发款会有确认数限制吗？是否可以0确认一直发？

可以零确认一直发

**Comment by doitian** [comment]
多账号转账和链式交易都是和 Bitcoin 相同的

**Comment by chaoticlonghair** [comment]
[How to Send CKB Transactions: N Inputs M Outputs (N>=1, M>=0)](https://github.com/nervosnetwork/ckb/tree/develop/rpc#send_transaction)

**Comment by wwaayyaa** [comment]
> 是指 ckb-cli 的 `transfer` 命令吗？

是的，我在使用这个命令转账。

**Comment by wwaayyaa** [comment]
> 一个 tx 里放多个 outputs 给不同的收款人就可以了，因为是一个交易不存在需要等确认。

可能一开始我方向不对，在使用cbx-cli server rpc，发现只有transfer。
如果自己构造交易，能否提供一个大致流程？
如何获取utxo, 是通过 get_live_cells_by_address 获取吗？然后是签名和提交交易。
才开始接触，有点云里雾里。

**Comment by wwaayyaa** [comment]
> [How to Send CKB Transactions: N Inputs M Outputs (N>=1, M>=0)](https://github.com/nervosnetwork/ckb/tree/develop/rpc#send_transaction)

Thanks. Regarding the previous steps, getting outputs, signatures, is there a document that specifically explains how to generate transactions? Thanks again.

**Comment by doitian** [comment]
如果用 cli 构建复杂交易可以看下 https://github.com/nervosnetwork/ckb-cli/wiki/Handle-Complex-Transaction
